scripts/init_indy_keys

```diff
@@ -10,7 +10,7 @@ from plenum.common.util import randomString
 from indy_common.config_util import getConfig
 
 config = getConfig()
-keepDir = config.baseDir
+keepDir = os.path.join(config.baseDir, config.NETWORK_NAME)
```

Comment:
Please make similar changes in `init_bls_keys` and `enable_bls`

Reply:
done